torch/masked/maskedtensor/core.py

```diff
@@ -0,0 +1,545 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+
+import warnings
+
+import torch
+from torch._masked import _sparse_coo_where, _sparse_csr_where
+from torch.overrides import get_default_nowrap_functions
+
+
+__all__ = [
+    "MaskedTensor",
+    "is_masked_tensor",
+]
+
+
+def is_masked_tensor(a):
+    r""" Returns True if the input is a MaskedTensor, else False
+
+    Args:
+        a: input MaskedTensor
+
+    Shape:
```

Comment:
Do we unconditionally give this for every function that takes Tensor input(s)? If not, I think it's unnecessary here.

Reply:
yeah that's right - i can just remove it then